templates/dropdown_form.html.twig

```diff
@@ -193,6 +193,12 @@
                             {% set fields_params = fields_params|merge({'types': config(field['itemtype_list'])}) %}
                         {% endif %}
                         {{ fields.dropdownItemTypes(field['name'], item.fields[field['name']], field['label'], fields_params) }}
+                    {% elseif type == 'registeredIDChooser' %}
+                        {% set chooser_field %}
+                            {{ call('RegisteredID::showAddChildButtonForItemForm', [item, '_registeredID', null, false])|raw }}
+                            {{ item.displaySpecificTypeField(item.fields['id'], field, fields_params) }}
```

Comment:
As this type is already handled in the `displaySpecificTypeField()` method, why not calling `RegisteredID::showAddChildButtonForItemForm()` from there ?